blocks/columns/columns.css

```
@@ -302,15 +347,34 @@
         gap: 80px;
     }
 
+    .content-band div:empty {
+        display: unset;
+    }
+
     .terms > div > div:nth-of-type(even) {
-        flex: 2;
-        margin-left: var(--spacer-layout-04);
+        flex: 1;
+        margin-left: unset;
+    }
+
+    .terms > div > div:nth-of-type(odd) {
+        width: 167px;
+        flex: unset;
     }
 
     .columns > div > div:first-of-type {
         margin-left: unset;
     }
 
+    .teaser-list-manual picture,
+    .teaser-list-manual p > picture {
+        display: unset;
+    }
+
+    .section:not(.leadspace) .image-end picture,
+    .section:not(.leadspace) .image-start picture {
```

Please update this to 

```suggestion
.columns.image-end picture,
.columns.image-start picture {
```
to match your change made above